lib/Elastica/Arrayable.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace Elastica;
+
+/**
+ * Interface for params.
+ *
+ *
+ * @author Evgeniy Sokolov <ewgraf@gmail.com>
+ */
+interface Arrayable
```

Comment:
I disagree. "Arrayable" is more used, for example https://github.com/search?l=php&p=1&q=Arrayable&ref=searchresults&type=Code&utf8=%E2%9C%93 and https://github.com/search?l=php&p=1&q=ToArrayInterface&ref=searchresults&type=Code&utf8=%E2%9C%93

Also Arrayable used in some frameworks, like Yii and seems something have Laravel API.

It is common practice to call such objects like "*able", that show to which one thing it can be transformed. Also there is already SearchableInterface. 

At least I will rename Nameable to NamebleInterface and Arrayable to ArrayableInterface for match naming convention.

For name it ToArrayInterface lets hear another opinions, because your one looks like a little bit personal.